Types/typeconfig.tsx

```diff
@@ -0,0 +1,22 @@
+//Navigation Types
+
+import { NativeStackScreenProps } from "@react-navigation/native-stack";
+
+export type StackScreens = {
+  Home: undefined;
+  Settings: undefined;
+};
+
+export type HomePropsType = NativeStackScreenProps<StackScreens, "Home">;
+export type SettingsPropsType = NativeStackScreenProps<
+  StackScreens,
+  "Settings"
+>;
+
+//Header Types
```

Comment:
Everything above this is not needed, neither is footerProp since there is no footer.
I missed it in the first PR